js/src/sdk/index.ts

```diff
@@ -45,6 +47,29 @@ export class Composio {
         this.actions = new Actions(this.backendClient);
         this.integrations = new Integrations(this.backendClient);
         this.activeTriggers = new ActiveTriggers(this.backendClient);
+
+        this.checkForLatestVersionFromNPM();
+    }
+
+    /**
+     * Checks for the latest version of the Composio SDK from NPM.
+     * If a newer version is available, it logs a warning to the console.
+     */
+    async checkForLatestVersionFromNPM() {
+        try {
+            const packageName = "composio-core";
+            const packageJsonDir = getPackageJsonDir();
+            const currentVersionFromPackageJson = require(packageJsonDir + '/package.json').version;
+        
+            const response = await axios.get(`https://registry.npmjs.org/${packageName}/latest`);
+            const latestVersion = response.data.version;
+
+            if (latestVersion !== currentVersionFromPackageJson) {
```

Comment:
Just a suggestion, we should use a greater than condition here instead of not equals. This will not break the state in case, one day we yank the latest release and the client has the latest version installed. 

```
var semver = require('semver');
semver.gt('1.2.3', '9.8.7') // false
```